code/API_definitions/age-vefification.yaml

```
@@ -0,0 +1,529 @@
+                  idDocument: 66666666q
+                  name: Federica Sanchez Arjona
+                  givenName: Federica
+                  familyName: Sanchez Arjona
+                  middleNames: Sanchez
+                  familyNameAtBirth: YYYY
+                  birthdate: '1978-08-22'                     
+                  email: "FedericaSanchez.Arjona@gmail.com" 
+
+      responses:
+        '200':
+          description: OK
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+
+          content:
+            application/json:
+
+              schema:
+                $ref: '#/components/schemas/KYC_ResponseBody'
+              examples:
+                KYC_200Example:
+                  value:
+                    ageCheck: true
+                    verifiedStatus: true
+                    identityMatchScore: 90
+        "400":
+          $ref: '#/components/responses/Generic400'
+        "401":
+          $ref: '#/components/responses/Generic401'
+        "403":
+          $ref: '#/components/responses/Generic403'
+        "404":
+          $ref: '#/components/responses/Generic404'
+        "406":
+          $ref: '#/components/responses/Generic406'
+        "415":
+          $ref: '#/components/responses/Generic415'
+        "422":
+          $ref: '#/components/responses/Generic422'
+        "429":
+          $ref: '#/components/responses/Generic429'
+        "500":
+          $ref: '#/components/responses/Generic500'
+        "503":
+          $ref: '#/components/responses/Generic503'
+        "504":
+          $ref: '#/components/responses/Generic504'
+
+components:
+  securitySchemes:
+    openId:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+
+
+  headers:
+    x-correlator:
+      description: Correlation id for the different services
+      schema:
+        type: "string"
+   
+  parameters:
+    x-correlator:
+      name: x-correlator
+      in: header
+      description: Correlation id for the different services
+      schema:
+        type: "string"
+        
+  schemas:
+    KYC_RequestBody:
+      description: "Requested information to be compare with customer personal information"
+      type: "object"
+      required:
+        - ageThreshold
+  
+      properties:
+
+        ageThreshold:
+         type: "integer"
+         minimum: 0
+         maximum: 120
+         description: age to be verified.
+
+        phoneNumber:
+          description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.
+      type: string
+      pattern: '^\+[1-9][0-9]{4,14}$'
+      example: "+123456789"

```

The indentation is wrong here, quick fix

```suggestion
          type: string
          pattern: '^\+[1-9][0-9]{4,14}$'
          example: "+123456789"
```